### DISCUSSION TITLE
ideia to damaged tanks

### DISCUSSION THREAD

**Discussion by idalx** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1667] [General]
What do you think that when a tank is damaged its texture will be a little different when we repair it with a blowtorch, something a little similar to desh armor
![Captura de Tela (333)](https://github.com/user-attachments/assets/5a5761cf-c4be-4c4f-83f9-77ff91ecd972)
![Captura de Tela (334)](https://github.com/user-attachments/assets/679cf1ec-c119-43b5-a7bc-c6e7e7192eee)
![Captura de Tela (332)](https://github.com/user-attachments/assets/1e4f14f5-29a5-496f-9f24-fb752472d65f)
me and my friends are thinking about it
what do you think about this?

**Comment by HollyEreTic** [comment]
and what does she look like?
